### ISSUE TITLE
[bug] CMD+Q doesn't close application on MacOS

### ISSUE THREAD

**Issue by Sytten** [tauri-apps/tauri#5579]
### Describe the bug

I updated to the latest v1.2.0 and the CMD+Q stopped working (it doesn't close the application anymore).
I will try to do more tests, but it doesn't seem like it's our code.

### Platform and versions

```
Environment
  › OS: Mac OS 11.6.3 X64
  › Node.js: 16.17.0
  › npm: 8.15.0
  › pnpm: 7.14.0
  › yarn: 1.22.15
  › rustup: 1.24.3
  › rustc: 1.61.0
  › cargo: 1.61.0
  › Rust toolchain: stable-x86_64-apple-darwin

Packages
  › @tauri-apps/cli [NPM]: 1.1.1 (outdated, latest: 1.2.0)
  › @tauri-apps/api [NPM]: 1.0.1 (outdated, latest: 1.2.0)
  › tauri [RUST]: git+https://github.com/caido/dependency-tauri?branch=caido#a93fd1d93998247d084c5a4454447c8d4b5d554b (1.2.0),
  › tauri-build [RUST]: git+https://github.com/caido/dependency-tauri?branch=caido#a93fd1d93998247d084c5a4454447c8d4b5d554b (1.2.0),
  › tao [RUST]: 0.15.0,
  › wry [RUST]: 0.22.0,

App
  › build-type: bundle
  › CSP: default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  › distDir: ../caido-ui/dist
  › devPath: http://localhost:10000/
  › framework: Vue.js

App directory structure
  ├─ dist
  ├─ node_modules
  ├─ scripts
  └─ src
```

**Comment by FabianLars** [comment]
Probably https://github.com/tauri-apps/wry/issues/749

**Comment by lucasfernog** [comment]
Fixed in tao v0.15.3. Please run `cargo update` in the `src-tauri` folder.

**Comment by tw93** [comment]
when i run `npm run tauri build -- --target universal-apple-darwin`, the bug still exists.
but when i run `npm run tauri build -- --target x86_64-apple-darwin` and m1 is ok
and `npm run dev` is ok
